### ISSUE TITLE
4.1.3 -> 4.1.4 breaking

### ISSUE THREAD

**Issue by marvinroger** [egoist/vbuild#69]
Hi (me again 😉)

greenkeeper is telling me 4.1.4 [broke my build](https://github.com/INTECH-RGBH/homie-dashboard/issues/16).

The problem is this commit: https://github.com/egoist/vbuild/commit/ef2e0788f19eb91f0d37d70ac409cadb5e557405

Broken build output: https://travis-ci.org/INTECH-RGBH/homie-dashboard/jobs/178998122#L400

The problematic code is:

```html

<style lang="sass" scoped>
  .image {
    margin: auto;
    margin-bottom: 10px;
  }
</style>
```

Thanks!

**Comment by egoist** [comment]
Yeah, it's a fix in `v4.1.4`, now indented style is `sass`,  for non-indented style it's `scss`

**Comment by marvinroger** [comment]
Fine! So my code was wrong in the first place, you did not actually break semver. My bad.
